browser/brave_news/brave_news_tab_helper.cc

```diff
@@ -212,9 +212,9 @@ void BraveNewsTabHelper::OnFoundFeedData(
   }
 
   if (feeds.empty()) {
-    rss_page_feeds_.erase(
-        base::ranges::remove(rss_page_feeds_, feed_url, &FeedDetails::feed_url),
-        rss_page_feeds_.end());
+    auto to_remove =
+        std::ranges::remove(rss_page_feeds_, feed_url, &FeedDetails::feed_url);
+    rss_page_feeds_.erase(to_remove.begin(), to_remove.end());
```

Comment:
Is it worth changing to `std::erase_if` as it is being touched?

Reply:
everywhere below also

Reply:
I think this is outside the scope of this change. I'm just migrating away of `base::ranges`, but I don't want to divert this PR into some sort of refactoring for these operations.

Reply:
sure, makes sense